Auth0/Telemetry.swift

```diff
@@ -100,7 +100,9 @@ public struct Telemetry {
     }
 
     static func swiftVersion() -> String {
-        #if swift(>=4.0)
```

Comment:
Quick change to add support for 5.x logging and no you can't simply ask for the literal string version